packages/connector-aliyun-dm/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "@logto/connector-aliyun-dm",
+    "version": "0.1.0",
+    "description": "Aliyun DM connector implementation.",
+    "main": "lib/index.js",
+    "author": "Logto Team",
+    "license": "MPL-2.0",
+    "private": true,
+    "scripts": {
+        "preinstall": "npx only-allow pnpm",
+        "precommit": "lint-staged",
+        "copyfiles": "copyfiles -u 1 src/**/*.md lib",
```

Comment:
we can just put md files inside a directory and include them in the package instead of copy them to `lib`?

Reply:
The current implementation follows @demonzoo 's design.
Do you mean only .md files related to connectors? We have some other .md files for quick-start guide.
We may add an issue to track and do this (gather all connector-related .md files).